documentation/docs/opal-plus/troubleshooting.mdx

```diff
@@ -0,0 +1,41 @@
+---
+sidebar_position: 10
+title: Troubleshooting
+---
+
+When something goes wrong, we are here to help.
+
+Feel free to reach out to us on Slack on our [community channel](https://bit.ly/opal-slack)
+or in your dedicated OPAL+ support channel, and we will do our best to assist you quickly.
+
+### Seeking Support
+
+If you are experiencing issues with OPAL+, please provide us with the following information:
+
+#### Extract Logs
+
+On production, we advise you to connect OPAL+ to your logging system to collect logs.
+Configure the [LOG_SERIALIZE](/getting-started/configuration) environment variable to `true` to serialize logs in JSON format.
+
+When running OPAL+ locally or in a development environment, you can extract logs from the console.
+Alternatively, you can enable logging to file by setting the [LOG_TO_FILE](/getting-started/configuration) to `true`.
+
+You can also enable debug logs by setting the [LOG_LEVEL](/getting-started/configuration) environment variable to `debug`.
+
+#### Export Configuration
+
+You can export your configuration by running the following command:
+
+```bash
+opal-client print-config
+```
+
+Make sure to censor any sensitive information, like passwords or API keys, before sharing the configuration.
+
+## Common Issues
+
+More common issues and their solutions are available in the [OPAL documentation](/getting-started/running-opal/troubleshooting).
+
+:::note
```

Comment:
You can add those 2 points 
Full description of the issue
A way to reproduce (e.g. a docker-compose or something) is optional and can help us reproduce and resolve this issue quicker